src/apps/shared/test/client.py

```diff
@@ -11,14 +11,14 @@ class TestClient:
     def __init__(self):
         app = create_app()
         self.client = AsyncClient(app=app, base_url="http://test.com")
-        self.headers = dict()
+        self.headers = {}
 
     @staticmethod
     def _prepare_url(url, query):
         return f"{url}{urllib.parse.urlencode(query)}"
 
     def _get_updated_headers(self, headers: dict | None = None) -> dict:
-        headers_ = dict(self.headers)
+        headers_ = self.headers
```

Comment:
here we have to work with copy of header to update, it may affect next request headers

Reply:
What about using `copy.copy()`